### ISSUE TITLE
Invalid unit when giving quarters to diff method

### ISSUE THREAD

**Issue by tiberiuzuld** [moment/luxon#776]
Hello,
I found an issue when passing `quarters` unit to the `diff` method:

```
luxon.DateTime.local().diff(luxon.DateTime.fromMillis(0), 'quarters')

duration.js:324 Uncaught Error: Invalid unit undefined
    at normalizeUnit (duration.js:324)
    at normalizeObject (util.js:259)
    at Function.fromObject (duration.js:245)
    at _diff (diff.js:70)
    at DateTime.diff (datetime.js:1760)
    at <anonymous>:1:24
```

**Comment by nigellima** [comment]
Hello. I just came across this error. Has this fix been released?

**Comment by tiberiuzuld** [comment]
Hi @nigellima ,
Yes it is fixed in v1.26.0
I think it was forgotten to be added to release notes.
The commit history shows that it was included in the release.
Also I just tested it myself in our application and works.

**Comment by nigellima** [comment]
@tiberiuzuld it was indeed fixed. I was using just one version before it hehe thank you very much
